Source/SelfService/Web/microservice/dataTable.tsx

```diff
@@ -28,25 +28,34 @@ const styles = {
     }
 };
 
-export const sortByRuntimeVersion = (params: GridValueGetterParams) => {
-    const runtimeVersion = params.row.edit.extra.runtimeImage.replace(/dolittle\/runtime:/gi, '');
+const sortByRuntimeVersion = (params: GridValueGetterParams) => {
+    const runtimeVersion = params.row.edit?.extra?.runtimeImage?.replace(/dolittle\/runtime:/gi, '');
+    if (runtimeVersion === undefined) return 'N/A';
+
     return `${runtimeVersion || 'None'}`;
 };
 
-const tooltipCell = (params: GridRenderCellParams) => (
-    // TODO: Tooltip needs public urls. Map them into title.
-    <Tooltip title={``} arrow>
-        <span>{params.row.edit.extra.isPublic ? 'Available' : 'None'}</span>
-    </Tooltip>
-);
+const tooltipCell = (params: GridRenderCellParams) => {
```

Comment:
Lets rename this to:
```suggestion
const publicUrlCell = (params: GridRenderCellParams) => {
```